parent/pom.xml

```diff
@@ -143,6 +153,13 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>org.infinispan</groupId>
+                <artifactId>infinispan-bom</artifactId>
+                <version>${version.infinispan}</version>
```

Comment:
This import is only used to inherit the version of ```org.infinispan:infinispan-client-hotrod``` and  ```org.infinispan:infinispan-core```, wouldn't be simpler to just use ```<version>${version.infinispan}</version>``` in those two dependencies?

Reply:
meh. I'll change it... It was just to test if the bom is okish :)